dotnetv3/dynamodb/scenarios/PartiQL_Basics_Scenario/PartiQL_Basics_Scenario/PartiQLBasics.cs

```diff
@@ -10,7 +10,7 @@
 // Separator for the console display.
 var SepBar = new string('-', 80);
 const string tableName = "movie_table";
-const string movieFileName = "moviedata.json";
+const string movieFileName = @"..\..\..\..\..\..\..\..\resources\sample_files\movies.json";
```

Comment:
The PartiQL program has a note that seems outdated: 

// Before you run this example, download 'movies.json' from
// https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GettingStarted.Js.02.html,
// and put it in the same folder as the example.


You may want to remove that or modify it.

Reply:
Removed.